primary/tests/nodes_bootstrapping_tests.rs

```diff
@@ -54,13 +52,13 @@ async fn test_node_staggered_starts() {
 
     tokio::time::sleep(node_staggered_delay).await;
 
-    let rounds = authorities_current_round(&cluster).await;
+    let rounds = authorities_latest_commit_round(&cluster).await;
     assert_eq!(
         rounds.len(),
         3,
-        "Expected to have received metrics from only three nodes"
+        "Expected to have received commit metrics from only three nodes"
     );
-    assert!(rounds.values().all(|v| v > &1.0), "We have only (f) unavailable nodes, so all should have made progress more than just the first round");
+    assert!(rounds.values().all(|v| v > &1.0), "We have only (f) unavailable nodes, so all should have made progress and committed at least after the first round");
```

Comment:
1. one of the authorities (since you have N-1) should report a last committed round of 0,
2. rather than `v > &1.0` I would go for `*v > 1u64`, which I think might work?

Reply:
Oooh, I figured out 1:
- when you look at the network from one authority's point of view among the 3, there is always one that's unreachable, and reports round=0,
- but this is not what we're doing here, rather `authorities_latest_commit_round` is polling each primary's metric among the ones that are up.

Reply:
Exactly, we only take into account the metrics from the nodes that are up and running - I mean from the others there is not reason to as they are not up yet.